src/live_restore/live_restore_server.c

```diff
@@ -152,16 +170,14 @@ __live_restore_worker_run(WT_SESSION_IMPL *session, WT_THREAD *ctx)
     /* FIXME-WT-13897 Replace this with an API call into the block manager. */
     WT_FILE_HANDLE *fh = bm->block->fh->handle;
 
-    __wt_verbose_debug3(
-      session, WT_VERB_FILEOPS, "Live restore worker: Filling holes in %s", work_item->uri);
+    __wt_verbose_debug1(
+      session, WT_VERB_LIVE_RESTORE, "Live restore worker: Filling holes in %s", work_item->uri);
```

Comment:
As I was understanding the code for progress it felt like this should be level 1. Push back if you disagree and I'll move it (and the one 2 lines below) back to level 3.

Reply:
This is by some definition a progress message. Do we define a progress messages as messages pertaining to the work on a single file?

In this case I would make these level 2 as they'll be fairly spammy but are useful messages for basic debugging.